packages/panzer/adapters-stk/src/responses/Panzer_STK_ResponseEvaluatorFactory_SolutionWriter_impl.hpp

```diff
@@ -104,8 +104,9 @@ buildAndRegisterEvaluators(const std::string& /* responseName */,
     allFields.insert(allFields.end(),physicsBlock.getProvidedDOFs().begin(),physicsBlock.getProvidedDOFs().end());
 
   // Add in tangent fields
-  if(addSolutionFields_)
-    allFields.insert(allFields.end(),physicsBlock.getTangentFields().begin(),physicsBlock.getTangentFields().end());
+  // TODO BWR IS THIS ACTUALLY SUPPORTED CORRECTLY?
+  //if(addSolutionFields_)
+  //  allFields.insert(allFields.end(),physicsBlock.getTangentFields().begin(),physicsBlock.getTangentFields().end());
```

Comment:
With the tangent fields now in the physics blocks, this caused issues with output